lib/experimental/new_embed.dart

```diff
@@ -245,81 +268,114 @@ class NewEmbed {
   }
 
   void _displayIssues(List<AnalysisIssue> issues) {
-    int errorCount = 0;
-    int otherCount = 0;
-
-    for (AnalysisIssue issue in issues) {
-      if (issue.kind == 'error') {
-        errorCount++;
-      } else if (issue.kind == 'info' || issue.kind == 'warning') {
-        otherCount++;
-      }
+    analysisResultBox.hide();
+    testResultBox.hide();
+
+    if (issues.isEmpty) {
+      return;
     }
 
-    if (errorCount == 0 && otherCount == 0) {
-      analysisResultBox.hide();
-    } else {
-      String message;
-      FlashBoxStyle style;
-
-      if (errorCount > 0 && otherCount > 0) {
-        message = 'Analyzer found $errorCount error${errorCount > 1 ? 's' : ''}'
-            ' and $otherCount other issue${otherCount > 1 ? 's' : ''}.';
-        style = FlashBoxStyle.error;
-      } else if (errorCount > 0) {
-        message =
-            'Analyzer found $errorCount error${errorCount > 1 ? 's' : ''}.';
-        style = FlashBoxStyle.error;
-      } else {
-        message =
-            'Analyzer found $otherCount issue${otherCount > 1 ? 's' : ''}';
-        style = FlashBoxStyle.warn;
+    List<String> messages = issues.map((AnalysisIssue issue) {
+      String message = issue.message;
+      if (message.endsWith('.')) {
+        message = message.substring(0, message.length - 1);
       }
+      return '$message - line ${issue.line}';
+    }).toList();
 
-      analysisResultBox.showStrings([message], style);
-    }
+    analysisResultBox.showStrings(messages, FlashBoxStyle.warn);
```

Comment:
There's also a showElements if you'd like to make them anchors that fire playground's jumpTo when clicked. I could also do that myself in a separate PR.

Reply:
If you don't mind updating after landing, that would be great.

Reply:
SGTM